widgets.py

```diff
@@ -475,4 +475,5 @@ def loadFilterSymbol(self):
     def zoomToFilter(self):
         if self.controller.hasValidFilter():
             iface.mapCanvas().setExtent(self.controller.currentFilter.geometry.boundingBox())
+            iface.mapCanvas().zoomByFactor(1.05)
```

Comment:
```suggestion
            iface.mapCanvas().zoomToFeatureExtent(self.controller.currentFilter.geometry.boundingBox())
```

I much prefer this approach, thank you very much for making me aware of that method :)
Adding as a new suggestion that I can easily commit because the removal of the `zoomByFactor` line was missing.